### DISCUSSION TITLE
Updating Lit dependencies in Vite

### DISCUSSION THREAD

**Discussion by StillOnMyWay** [lit/lit#2111] [Ideas]
Hi all. Are there any plans to submit a PR to Vite.js' quickstart to update the Lit dependencies to use lit instead of lit-element^2.4.0? It's something I had considered, but I wasn't sure if the fact that Lit is still in rc-release candidate phase was a reason not to.

Vite is vastly gaining popularity, way more than open-wc's project creator, so I'm thinking it might be best to get people started the right way. 

https://github.com/vitejs/vite/tree/main/packages/create-vite

**Comment by graynorton** [comment]
I would say it's not too early to submit a PR. Lit 2.0 is definitely stable enough at this point (targeting an official production release within the month). At a glance, it looks to me like the most sensible approach would be to add new `template-lit` and `template-lit-ts` templates that use Lit 2.0, allowing the existing `template-lit-element` and `template-lit-element-ts` templates to be deprecated and eventually removed—but I'd suggest consulting with the folks who originally created those templates before proceeding. You'll probably find them in the Vite Discord.

**Reply by StillOnMyWay** [reply]
That's a great, great point, re: add new instead of remove.  I'm going to have to plan that PR out
